src/plugins/crypto/README.md

```diff
@@ -109,34 +110,35 @@ Set the CMake variables `OPENSSL_INCLUDE_DIR` and `OPENSSL_LIBRARIES` to your de
 
 The crypto plugin will encrypt and decrypt values using AES-256 in CBC mode.
 
-The key derivation is still WIP.
+The GPG binary is required for handling the cryptographic keys.
 
-### Planned Features ###
+At the moment the plugins will only run in a UNIX/Linux-like system.
```

Comment:
Maybe a bit more specific. Do you need something beyond POSIX (and the gpg binary)?